go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/ava-labs/awm-relayer
 
-go 1.21.10
+go 1.22.3
```

Comment:
Lets keep this at 1.21.10. We always aim to use the latest patch version of the minor version one behind latest.

Reply:
Sorry I missed your other comment explaining the need for the change here. Would still prefer to keep our golang version consistent across the board for now. The motivation to stay a minor version behind is avoid the increased risk of bugs in the latest version

Reply:
Reverted back to go 1.21.10